tests/integration/utils.py

```diff
@@ -94,12 +96,12 @@ def pull_image(self) -> None:
 
     def run_cmd_on_image(
         self,
-        cmd: List,
-        tmpdir: StrPath,
+        cmd: list[str],
+        tmp_path: Path,
```

Comment:
This hunk (multiple occurrences) seems like a non-related cosmetic change.

Reply:
This hasn't been addressed. Again, we're close to merging so let's not make a deal out of this, but please make sure you perform cosmetic changes like these (if at all) in a separate commit to avoid polluting a diff with unrelated changes, it just prolongs the review itself.